lib/dataset/test/concat_test.cpp

```diff
@@ -287,26 +299,6 @@ TEST(ConcatenateTest, concat_2d_coord) {
   EXPECT_EQ(abba, expected);
 }
 
-TEST(ConcatenateTest, dataset_with_no_data_items) {
-  Dataset ds;
-  ds.setCoord(Dim::X,
-              makeVariable<double>(Dims{Dim::X}, Shape{4}, Values{1, 2, 3, 4}));
-  ds.setCoord(Dim("points"), makeVariable<double>(Dims{Dim::X}, Shape{4},
-                                                  Values{.1, .2, .3, .4}));
-  EXPECT_EQ(concat2(ds.slice({Dim::X, 0, 2}), ds.slice({Dim::X, 2, 4}), Dim::X),
-            ds);
-}
-
-TEST(ConcatenateTest, dataset_with_no_data_items_histogram) {
-  Dataset ds;
-  ds.setCoord(Dim("histogram"), makeVariable<double>(Dims{Dim::X}, Shape{4},
-                                                     Values{.1, .2, .3, .4}));
-  ds.setCoord(Dim::X, makeVariable<double>(Dims{Dim::X}, Shape{5},
-                                           Values{1, 2, 3, 4, 5}));
-  EXPECT_EQ(concat2(ds.slice({Dim::X, 0, 2}), ds.slice({Dim::X, 2, 4}), Dim::X),
-            ds);
-}
-
```

Comment:
Right, such datasets were not allowed at some point. I have since added `
TEST_F(Concatenate1DTest, empty_dataset)` further up in the same file. I'm also going to add one for histograms.